test/unit/cloudinaryConfig.test.js.mv

```diff
@@ -0,0 +1,17 @@
+import cloudinary from 'cloudinary-core';
+import VideoPlayer from '../../src/video-player';
+import window from 'global/window';
+import videojs from '../../node_modules/video.js';
+// import defaults from '../../src/config/defaults.js';
+const cld = cloudinary.Cloudinary.new({ cloud_name: 'demo' });
+cloudinary.VideoPlayer = VideoPlayer;
+
+
+describe('secure true test', () => {
+  it('test force secure true', () => {
+    jest.useFakeTimers();
+    document.body.innerHTML = '<div><video id="test"/></div>';
+    let vp = new VideoPlayer('test', { hideContextMenu: true }, false);
+    console.log(vp);
```

Comment:
@yaniv-li can you please add a test that checks the source is https
and a test that checks that when passing secure=false to cloudinary, it is http?

Reply:
@maoznir I added explicit secure true **in the player options** not in the cloudinary options  
there are 2 tests for default  and explicit false